aim/aim_store.py

```diff
@@ -317,6 +318,15 @@ def resource_to_db_type(self, resource_klass):
         return self.db_model_map.get(resource_klass)
 
     def add(self, db_obj):
+        if 'object_dict' in db_obj.__dict__:
```

Comment:
I think I understand what you try to do here but can you put some comments to explain it? This will help ppl in the future to read or maintain it.

Reply:
oh yes, i will add comments

Reply:
working on this

Reply:
I see this still isn't in the same place. Did you look into moving this?